lambdas/db-migration/tsconfig.json

```diff
@@ -1,11 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "outDir": "dist/lambda",
-    "declaration": false,
-    "declarationMap": false,
-    "sourceMap": false,
```

Comment:
Odd. 

This change breaks the quick and dirty CL invocation:

node -e "require('./dist/lambda/index').handler({ command: 'latest'})"

Reply:
I'm reverting 9dad727ebd30ec28eac966ac1573d66a9c9c14c9 as it doesn't work.

Reply:
Not that it means in any way we don't need to sort the issue.